web/src/popover_menus.js

```diff
@@ -1159,6 +1159,19 @@ export function initialize() {
     register_popover_menu("#personal-menu", {
         placement: "bottom",
         offset: [-50, 0],
+        // The stratergy: "fixed"; and eventlistener modifier option ensures that the personal menu does not
+        // modify its position or disappear when user zoom the page.
+        popperOptions: {
+            strategy: "fixed",
```

Comment:
This commit looks good except "strategy" is mispelled in the comment and the comment isn't line-wrapped properly. I think you should squash it into the main commit though.

Reply:
Done